codex-ui/src/vue/row/RowList.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <FormSection :title="listName">
+    <slot></slot>
+  </FormSection>
+</template>
+
+<script lang='ts' setup>
+import FormSection from '../form/FormSection.vue';
+
+defineProps<{
+  /**
+   * The list name
+   * Displayed above all listed items
+   */
+  listName: string;
+}>();
+</script>
+
+<style module lang="postcss">
+
+</style>
```

Comment:
Accourding to mockups, RowList should have background and rounded corners

![image](https://github.com/codex-team/notes.web/assets/31101125/5bbe31df-d30c-4591-ab91-1f81e2902a71)